Klarna.Rest/SampleProjects/KlarnaCheckoutWebApp/README.md

```diff
@@ -0,0 +1,27 @@
+ASP.NET [MVC Web App](https://dotnet.microsoft.com/apps/aspnet/mvc) Sample Project
```

Comment:
Let us run this README through prettier [Link to result](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBBAygBQHQDkCiAKgAQDaAsgGoDCxA6nAEbGoAOrAugBQAWMMrAM5IA9CIAmEGFDgxsAWwCWYAE4RBEAGZzI8kQEN2gg4NYyYI+QDcwASmLp981gBs4xTGoBWcMDAA6UAC8IaFh4RGRUdGBgcSEPO4YOAQkVLSCTq7urN6+MMTicPLQgjAq+vCCxDwQAO7EMBDEdRAqANYtijA8jYnEANIu+ipQ+sTUiWDtEACuBfpQ4oPDo+OY+gCe8ggwgtixUMQAxMfEAEqzR54QPn6HxACqgu4Akq-oxC8qVnAqxG1iJRFIJZvoXA4YLNxIpmk1iCorn13OgIC55rCoNhiMQHgAqPG4KRwAlIYgAMTRLnqyLI5GUag02mIAHNZooii5FDJuHwBMIxJIwPslKp1FodBA9AgALSzYxWEFglxlaGwkQcuAiACO7OmZRGMBl+lM5hlkBUcAA-Iq4HUglZBDKAEwABgAjABOWyBeGMdkuZaLZaIo49HJ5PwHKBxBIR275L5-X7VPmsUQialgcG1MpIACsrqLxFYbQKjE2hTgmn0sxccgepzONzuBWolqKsEU4MEDwAmnNiDI4Mt4WAeIsWe5w8RVCPdt2VcQuPK-mMdsRgyWTYJWipxPZubSAAYACSlcGo0HKVL+2GFx+ImkUbkazUY7n0jFfTUCU4KM6TL4MzzF8UCKOwsibksCIIEU-wzhs2y7E+bTyNijZnAAIsUzRbuccCgvWvYxkcqDaH8CJXOBUAsrSuQJn4xCbIOdQvhCz4wTArTEHAAAeWSvpo1J1MIxAEkMIxjBMUwgTABLQcsEmrNJSE7LABLRtQXLTLS-ggOgcDuPxglwPpxD+vw0Bvkm079EUJTblO0ZxAAhKQ3Amqw2DmAA+tYYDYGYLI+lAMoyoE7ncO0rC+Q5EBBbRtggAANCAECsDAmKCMgoAjGodQbJaUA5Sg+hWBAHKpSAjAVNMsjoKw+hgNyLLIOUsxwGlfDyC4dA8N0hFNWAcCGSV3SKIqMCbMg4CCDlaXct8MCePoLLyPoyA1iqXUgF4gh8QAQnV7QNU4cAADLcnAW09rt+18egrVuAAirMxK3TtaVNSo3yzRtHSSHUUDVbk3IwHQHI9MgAAcrrfYycB0BUrCzbkhHJjdaW6sSNxCLNJoysORTiNVlq6oolqretm1INtLxpS8SjtYiu2CM9cBvR9dN3WlMBfpD4jQ0gzp8xUL6tVe8gbbNUDQFjICroQX6lfTcAAL7q0AA)

It is a lot cleaner afterward. Also, I do not want to introduce it here in a .NET repository, so let us run it manually.

Reply:
Hm, it actually changed not to much. I don't know how you get the initial version of the text you pasted to the service, but the original README file has a different intentions and new lines.

The only difference I see - the service replaced the "=====" with "#" (I don't like this change, because the ==== in most cases means the top 1 header, like the `title` of the file)

And the second change - `## ![](asp.net_mvc.png)`. Well, the service did an image to be a headers.... And also - I had a horizontal line between images in case to make a visual split between them, but the services removed this line and now the rendered result looks much worse.

Reply:
1) Prettier is formatting by https://commonmark.org/ standard - I think we should follow a standard and not opinions actually to reduce discussions and so on. Prettier wrote about this topic a lot on their 'Why Prettier?' site: https://prettier.io/docs/en/why-prettier.html So will not discuss this here, I am fine with your reasoning

2) GitHub Markdown says you should use `---` or `___` for a horizontal divider (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#horizontal-rule) and also have an empty line before and after it (the line break a lot of people ignore anyway and results into misrenderings a lot of time) - prettier is a good tool for catching those things early actually

Reply:
Fixed the line